CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Show permission error for accessing Telegram as Viewer ([1273](https://github.com/grafana/oncall/issues/1273))
 
+### Changed
+
+- Remove OSS_INSTALLATION env var in favour of LICENSE
```

Comment:
I wonder if we want to include this into the changelog, given the PR doesn't make any "visible" changes to users?

Reply:
+1